### DISCUSSION TITLE
I can't log in to my Facebook account

### DISCUSSION THREAD

**Discussion by dhrubo11** [Android-App-Patches/build-apps#88] [General]
![Screenshot_2024-02-07-14-11-25-293_com.facebook.katana.jpg](https://github.com/revanced-apks/build-apps/assets/17109342/c1c3e4c9-188b-40af-8751-dad50b2e4893)

**Comment by rabilrbl** [comment]
Unfortunately, I don't use facebook and have no Idea about this.

Join our community at https://t.me/revanced_apks_chat for further discussion on this topic
